h/static/scripts/annotator/guest.coffee

```diff
@@ -33,11 +31,9 @@ module.exports = class Guest extends Annotator
     ".annotator-hl mouseover":           "onHighlightMouseover"
     ".annotator-hl mouseout":            "onHighlightMouseout"
 
-  # Plugin / Options configuration
   options:
-    TextHighlights: {}
```

Comment:
No. Sorry that was a confusing diff. That plugin hasn't existed for a while, since no-dtm maybe, and this line had no effect.